po/fr.po

```diff
@@ -1799,7 +1796,7 @@ msgid "Mark as Finished"
 msgstr "Marquer comme terminé"
 
 #: src/queueloader.cpp:127
-#, fuzzy, c-format
+#, c-format
```

Comment:
I can't mark the offending line, but "podboat" in the translated message should start with a capital letter.